frontend/cypress/integration/featureFiles/graph_side_panel.feature

```diff
@@ -33,3 +32,42 @@ Feature: Kiali Graph page - Side panel menu actions
       | request_routing      |
       | fault_injection      |
       | request_timeouts     |
+
+  @skip
+  @multi-cluster
+  Scenario: Actions in kebab menu of the side panel for a service node with existing traffic routing
+    And there is a traffic routing for the "reviews" service present
+    And user clicks the "reviews" service node
+    And the side panel links should contain a parameter related to cluster name
+    And user opens the kebab menu of the graph side panel
+    And user clicks the "delete_traffic_routing" item of the kebab menu of the graph side panel
+    Then user should see the confirmation dialog to delete all traffic routing
```

Comment:
What if add nore steps to check the deletion as well?
"And user confirms the deletion dialogue
Then users sees no traffic"

Reply:
Good idea, will add the steps.

Reply:
thanks